src/extensions/renderer/canvas/drawing-images.js

```diff
@@ -23,9 +23,11 @@ CRp.drawInscribedImage = function( context, img, node, index ){
   var nodeY = pos.y;
   var fit = getIndexedStyle( node, 'background-fit', 'value', index );
   var repeat = getIndexedStyle( node, 'background-repeat', 'value', index );
-  var paddingX2 = node.pstyle( 'padding' ).pfValue * 2;
-  var nodeW = node.width() + ( getIndexedStyle( node, 'background-width-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
-  var nodeH = node.height() + ( getIndexedStyle( node, 'background-height-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
+  var nodeW = node.width();
+  var nodeH = node.height();
+  var paddingX2 = util.computePaddingValues(nodeW, nodeH, node.pstyle( 'padding' ), node.pstyle( 'padding-relative-to' ).value ) * 2;
+  var nodeTW = nodeW + ( getIndexedStyle( node, 'background-width-relative-to', 'value', index ) === 'inner' ? 0 : paddingX2 );
```

Comment:
I forgot the `node.padding()` function existed. However, that function doesn't exactly solve the problem. Here is the definition of the padding function above:

```
elesfn.padding = function(){
  var ele = this[0];
  var _p = ele._private;
  if( ele.isParent() ){
    ele.updateCompoundBounds();

    if( _p.autoPadding !== undefined ){
      return _p.autoPadding;
    } else {
      return ele.pstyle('padding').pfValue;
    }
  } else {
    return ele.pstyle('padding').pfValue;
  }
}
```
As you can see, it does not make special allowances for percentage padding values. However, what I can do is update the padding function to use `computePaddingValues(nodeW, nodeH, node.pstyle( 'padding' ), node.pstyle( 'padding-relative-to' ).value )` which should solve issue, and then go ahead and use `.padding()` for everything.